scripts/superdesk/editor2/editor.ctrl.js

```diff
@@ -352,7 +353,7 @@ function SdTextEditorController(_, EMBED_PROVIDERS, $timeout, $element, editor,
         * Compute an id for the block with its content and its position.
         * Used as `track by` value, it allows the blocks to be well rendered.
         */
-        generateBlockId: function(block) {
+        generateBlockId: function(block, i) {
```

Comment:
Is `i` an ID? If so, can you please call it `id`? It would be nice to update the documentation to specify what all the function arguments are.